src/main/java/com/room/hotelreservation/controller/CustomerController.java

```diff
@@ -1,28 +1,35 @@
-package com.room.hotelreservation.controller;
-
-import com.room.hotelreservation.model.request.CustomerRegistrationRequest;
-import com.room.hotelreservation.model.response.CustomerResponse;
-import com.room.hotelreservation.service.CustomerService;
-import jakarta.validation.Valid;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
-
-@RestController

```

Comment:
Add Documentation

Reply:
@pawankumaar added the documentation as per your review comments. Please verify